packages/kit/src/runtime/server/page/load_data.js

```diff
@@ -58,92 +59,98 @@ export async function load_server_data({ event, state, node, parent }) {
 		disable_search(url);
 	}
 
-	const result = await node.server.load?.call(null, {
-		...event,
-		fetch: (info, init) => {
-			const url = new URL(info instanceof Request ? info.url : info, event.url);
+	const load = node.server.load;
 
-			if (DEV && done && !uses.dependencies.has(url.href)) {
-				console.warn(
-					`${node.server_id}: Calling \`event.fetch(...)\` in a promise handler after \`load(...)\` has returned will not cause the function to re-run when the dependency is invalidated`
-				);
-			}
-
-			// Note: server fetches are not added to uses.depends due to security concerns
-			return event.fetch(info, init);
-		},
-		/** @param {string[]} deps */
-		depends: (...deps) => {
-			for (const dep of deps) {
-				const { href } = new URL(dep, event.url);
+	const result =
```

Comment:
this is really adding a lot of indentation. I wonder if it wouldn't be better to just do an early return when `!load`